releases/release-6.1.3.md

```diff
@@ -0,0 +1,94 @@
+---
+title: TiDB 6.1.3 Release Notes
+---
+
+# TiDB 6.1.3 Release Notes
+
+发版日期：2022 年 xx 月 xx 日
+
+TiDB 版本：6.1.3
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v6.1/quick-start-with-tidb) | [生产部署](https://docs.pingcap.com/zh/tidb/v6.1/production-deployment-using-tiup) | [下载离线包](https://cn.pingcap.com/product-community/?version=v6.1.3#version-list)
+
+## 提升改进
+
```

Comment:
- Enable transaction split and disable safeMode of a changefeed in TiCDC by defult. [#7505](https://github.com/pingcap/tiflow/issues/7505)
 - Improve kakfa protocol encoder performance.[#7540](https://github.com/pingcap/tiflow/issues/7540)[#7532](https://github.com/pingcap/tiflow/issues/7532)[#7543](https://github.com/pingcap/tiflow/issues/7532)